.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,53 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to use for this release. Please use https://semver.org.'
+        required: true
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Build release distribution"
+        run: make release
+        env:
+          FORCE_MOVE_RELEASE_TAG: "YES"
+          GITHUB_ACTOR: ${{ github.actor }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Commit and push updated Swift package description"
+        run: |
+            git add Package.swift
+            git commit -m "Update Package.swift for version ${{ github.event.inputs.tag }}"
+            git push
+
+      - name: "Draft a release"
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          name: "Sparkle ${{ github.event.inputs.tag }}"
+          tag_name: ${{ github.event.inputs.tag }}
```

Comment:
I didn't actually know you could pass an input. Does this get used or still WIP? move_tag() seems to get message from an existing tag.

Reply:
Never mind, I see it used above.